packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import '@cerner/terra-graphs/lib/components/Bar/BarGraph.module.scss';
```

Comment:
The terra-graphs package won't be release when we pull terra-grachs-docs into terra-ui, so any references to terra-graphs will fail.

Reply:
I created a new issue to address this:  #202